blebox: update lightBox and lightBox PRO device documentation (#35337)
home-assistant/home-assistant.github.io: source/_integrations/blebox.markdown

### wLightBox

This integration adds the Blebox device to Home Assistant as:

- multiple MONO lights entities,
- 1 or 2 lights CCT entities,
- 1 light RGB or RGBW or RGBCCT entity.

#### Key supported features

- On
- Off
- Effects
- Brightness
- Color (RGB, RGBW, RGBCCT mode only)
- White temperature control (RGBCCT & CCT mode only)
- Effects

#### Additional features

- Option to change control mode (linear / gamma correction) is available in wBox app.
- Option to change color mode (MONO/CCT) is available in wBox app.

> - Own effects are possible to create. Creator of effects is available in wBox app.
> - After settings change, device needs to be reloaded.

Reviewer: ```suggestion
- You can create your own effects. Creator of effects is available in wBox app.
- After a settings change, the device needs to be reloaded.
```

### wLightBox PRO

This integration adds the Blebox device to Home Assistant as:

- multiple MONO lights entities,
- 1 or 2 lights CCT entities,
- 1 light RGB or RGBW or RGBCCT entity.

#### Key supported features

- On
- Off
- Effects
- Brightness
- Color (RGB, RGBW, RGBCCT mode only)
- White temperature control (RGBCCT & CCT mode only)
- Effects

#### Additional features

- Option to change control mode (linear / gamma correction) is available in wBox app.
- Option to change color mode (MONO/CCT) is available in wBox app.

> - Own effects are possible to create. Creator of effects is available in wBox app.
> - After settings change, device needs to be reloaded.

Reviewer: ```suggestion
- You can create your own effects. Creator of effects is available in wBox app.
- After a settings change, the device needs to be reloaded.
```
